packages/libs/error-stack/src/lib.rs

```diff
@@ -402,25 +402,20 @@
 //! [`Result`] directly, but also a lazy variant that receives a function which is only called if
 //! an error happens.
 //!
-//! In addition to [`ResultExt`], this crate also comes with [`FutureExt`] (enabled by the
-//! `futures` feature flag), which provides the same functionality for [`Future`]s.
-//!
-//! Adding adaptors for [`Iterator`] and [`Stream`]  is a considered feature and can be prioritized
-//! depending on demand.
+//! In addition to [`ResultExt`], this crate also comes with [`FutureExt`], which provides the same
+//! functionality for [`Future`]s.
 //!
 //! [`Future`]: core::future::Future
-//! [`Stream`]: futures_core::stream::Stream
 //!
 //! ### Feature Flags
 //!
-//!  Feature   | Description                                                    | implies | default
-//! -----------|----------------------------------------------------------------|---------|--------
-//!  `std`     | Enables support for [`Error`] and, on nightly, [`Backtrace`]   |         | enabled
-//! `spantrace`| Enables the capturing of [`SpanTrace`]s                        |         | disabled
-//!  `futures` | Provides a [`FutureExt`] adaptor                               |         | disabled
-//!  `anyhow`  | Provides conversion from [`anyhow::Error`] to [`Report`]       |         | disabled
-//!   `eyre`   | Provides conversion from [`eyre::Report`] to [`Report`]        |         | disabled
-//! `pretty-print` | Provide color[^color] and use of unicode in [`Debug`] output |     | enabled
+//!  Feature       | Description                                                        | default
+//! ---------------|--------------------------------------------------------------------|----------
+//! `std`          | Enables support for [`Error`] and, on nightly, [`Backtrace`]       | enabled
+//! `pretty-print` | Provide color[^color] and use of unicode in [`Debug`] output       | enabled
+//! `spantrace`    | Enables the capturing of [`SpanTrace`]s                            | disabled
+//! `anyhow`       | Provides conversion from [`anyhow::Error`] to [`Report`]           | disabled
```

Comment:
Should we say provides conversion _through_ some mechanism? Like is it through `into_report` or some `From` implementation or?

Reply:
The trait `IntoReportCompat`, which provides `into_report()`, is implemented for `Result<T, eyre::Report>` and `Result<T, anyhow::Error>`.

Reply:
I think we should update the description then to say

> `Provides `into_report` to convert [`anyhow::Error`] to [`Report`]

(I'd suggest a change but there are two lines and I only commented on one 🙃)

Reply:
Applied suggestions in c845231da303469eb52123636225aba610ec3b06, thank you!